src/css/settings.css

```diff
@@ -1,38 +1,308 @@
-button {
-    background-color: #0060df;
+html, body {
```

Comment:
I don't think this file needs changes.

Reply:
Oops sorry about this, I don't know why I pushed this change. I'll remove it right away.